pyproject.toml

```diff
@@ -13,6 +13,9 @@ uvicorn = "^0.23.2"
 pydantic = "^2.4.2"
 pydantic-settings = "^2.0.3"
 httpx = "^0.25.0"
+sqlalchemy = "^2.0.22"
+py-healthcheck = "^1.10.1"
+psycopg2-binary = "^2.9.9"
```

Comment:
Consider using psycopg3 as it has support for async operations.

Reply:
Switched to psycopg3 in [ccc4f11](https://github.com/openearthplatforminitiative/flood-api/pull/1/commits/ccc4f11a9a5275abda79ea64ffc11c9332e81e63). Please note that psycopg3 is packaged as psycopg (there is no psycopg3 package)